lib/gibbon/api_request.rb

```diff
@@ -135,7 +135,7 @@ def rest_client
         faraday.response :raise_error
         faraday.adapter adapter
         if @request_builder.debug
-          faraday.response :logger, ::Logger.new(STDOUT), bodies: true
+          faraday.response :logger, builder.logger, bodies: true
```

Comment:
It should, I forgot to push one commit. Here is the fix: https://github.com/amro/gibbon/pull/216